src/zui/ZUITimeline/index.tsx

```diff
@@ -46,7 +48,11 @@ const ZUITimeline: React.FunctionComponent<ZUITimelineProps> = ({
     <Fade appear in timeout={1000}>
       <Grid container direction="column" spacing={5}>
         <Grid item>
-          <TimelineAddNote disabled={disabled} onSubmit={onAddNote} />
+          <TimelineAddNote
+            disabled={disabled}
+            onSubmit={onAddNote}
```

Comment:
If the state is moved in here, then this line could instead be:

```suggestion
            onSubmit={onAddNote}
            onSubmit={(note) => {
                try {
                    onAddNote(note);
                } catch (err) {
                    // Set state here
                }
            }
```

Reply:
Good suggestion, I've pushed a change where I moved the error handling